tests/installation/setup_zdup.pm

```diff
@@ -61,6 +61,11 @@ sub run {
         }
 
     }
+    if (is_leap('>15.2')) {
```

Comment:
I recommend to really move some explanation into the source code as it's far from obvious why that code is there

Reply:
the explanation added.